docs/ui.rst

```diff
@@ -0,0 +1,94 @@
+==============
+User Interface
+==============
+
+Thebe includes support for some built-in user interface elements.
+
+The aim is to provide simple elements that can be added to a page a styled accordingly by the consumer without neeing to [re]write coont code.
+Using these UI elements in the way shown here, assumes that you already have Thebe statically loaded on page.
+
+Built in UI elements are:
+
+- Activate Button
+- Kernel Status Widget
+
+Activate Button
+===============
+
+Configure this UI element as follows.
+
+Add a :code:`div` element to the page in the desired location.
```

Comment:
```suggestion
Add a ``div`` element to the page in the desired location.
```

I think double backticks does the same thing in rST (we should really convert this to MyST one day 😬 )